### ISSUE TITLE
[A11Y] Very low contrast ratio between asterisk (required fields) and background color

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9504]
## Issue Description
Some input and text fields in Login and Register User forms use an asterisk (*) to indicate that the control is required.
The current color to this element doesn't provide enough contrast ratio and may not be noticeable by low vision users.

## How to reproduce / Current behavior
1. Access a journal
2. Click on "Login" or "Register"
3. See the fields with asterisk
4.  Check the color contrast ratio using a color check, e.g.: WAVE Evaluation tool . It returns 3.46:1 and it should be 4.5:1

## What application are you using?
 OJS version 3.3.0

## Stack used 
- Device: Laptop
- OS: MacOS
- Browser: Firefox / Chrome / Edge

## Testing method
- Method: Automated tool - WAVE

## Action
Increasing the color ratio can be achieved while decreasing the lightness level of the current color (#ff4040).

### Results
![Image](https://github.com/pkp/pkp-lib/assets/4992902/aae43dce-1361-4c7d-9909-85ebef34be0f)

![Image](https://github.com/pkp/pkp-lib/assets/4992902/811ab689-52d7-4414-a1a4-95686b93ad5f)

**Comment by israelcefrin** [comment]
Hi @asmecher ,

I've created a PR that fixes the low contrast ratio for the element and the color used:
https://github.com/pkp/ojs/pull/4100

This issue is related to OJS 3.3 only.

**Comment by asmecher** [comment]
Merged, thanks, @israelcefrin!
